javascript/javascript3/week2/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0px;
+  background-image: linear-gradient(to right, #decba4, #3e5151);
+}
+h1 {
+  width: 100%;
+  text-align: center;
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  letter-spacing: 1px;
+  padding: 0 0 40px;
+  border-bottom: double #555;
+  text-shadow: 1px 1px 2px #3e5151;
+}
+.formOne,
+.formTwo {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.button_block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 20px 0px 20px 0px;
+}
+select {
```

Comment:
In general you should not use element selectors in your css unless you want to target all select elements everywhere in the the page or overwrite the browser styling. In case of this simple calculator it is ok to do s, but as complexity grows it migt be better to use classes.

Reply:
yeeah, I was to lazy in the end of coding :D